accel_code/excel_lookup.py

```diff
@@ -13,8 +14,8 @@ def excel_lookup(lab_id, date, excel_file):
     
     # convert all ID's to strings
     df.LabID = df.LabID.astype(str)
-    sess = df[( df['LabID'].str.contains(sub_id) ) & ( df['File date'] == date )]
-    sess_id = str(sess['LabID']).split('_')[1]
-    project = str(sess['Project'])
+    sess = df[( df['LabID'].str.contains(lab_id) ) & ( df['File date'] == date )]
+    sess_id = str(sess['LabID'].values[0].split('_')[1])
+    project = str(sess['Project'].values)[2:-2]
```

Comment:
depending on the project, the session information will be contained in either the project or the lab_id, we will need some logic to see which of the entries has an underscore and then extract session from either `LabID` or `Project` depending on which entry contains an underscore.